.github/workflows/testing.yml

```diff
@@ -224,3 +275,80 @@ jobs:
         run: |
           cat tarantool.log || true
           kill $(cat tarantool.pid) || true
+
+  run_tests_with_install_windows:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.repository)
+
+    runs-on: windows-2022
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        tarantool:
+          - '2.10.0.g0a5ce0b9c-1'
+        python:
+          - '3.10'
```

Comment:
Are there any restrictions on the python version on Windows? Does it need to be documented?

Reply:
Well, there are no restrictions, it's more like "test only minimal required" since Windows pipelines are rather long.

Reply:
I think then you need to add 3.6 (3.6, 3.10) since we claim it as a minimal supported version.

Reply:
That's a good comment, added